README.md

```diff
@@ -58,10 +54,10 @@ tricky, and requires some knowledge about the platform to make sure it won't bre
 
 So, if you're serious about getting Astarte in your production environment, you might want to learn
 more about it first. Start by [having a look at its
-architecture](https://docs.astarte-platform.org/astarte/latest/001-intro_architecture.html) and
-[finding out how it works](https://docs.astarte-platform.org/astarte/latest/001-intro_user.html). Once
+architecture](https://docs.astarte-platform.org/astarte/1.2/001-intro_architecture.html) and
+[finding out how it works](https://docs.astarte-platform.org/astarte/1.2/001-intro_user.html). Once
 you feel confident, head over to the [Administration
-Manual](https://docs.astarte-platform.org/astarte-kubernetes-operator/latest/001-intro_administrator.html).
+Manual](https://docs.astarte-platform.org/astarte-kubernetes-operator/22.11/001-intro_administrator.html).
```

Comment:
It is better to always point to the latest Operator doc, so that incompatibilities between Astarte and Operator are not present